promscale/visualize-data/jaeger.md

```diff
@@ -0,0 +1,38 @@
+# Connect Promscale and Jaeger Query
+This section shows you how to connect Promscale with
+[Jaeger][grafana-jaeger-datasource].
+
+Before you begin, you need the IP address of your Promscale instance. You can
+find this at the command prompt, using this command:
+```bash
+docker inspect <PROMSCALE_CONTAINER_NAME>
+```
+
+The IP address is listed in the `NetworkSettings → Networks → IPAddress`
+section.
+
+Alternatively, you can set the URL as `promscale:9202` (here `9202` is the gRPC endpoint of Promscale), where `promscale` is the name of the container.
```

Comment:
Again, this is very specific to Docker and I think this doesn't work if you run on Kubernetes.